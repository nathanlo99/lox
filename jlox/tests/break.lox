
var start = clock();
for (var i = 0; i < 20; i = i + 1) {
  print("The ceiling of square root of " + i + " is: ");
  for (var j = 0; j <= i + 1; j = j + 1) {
    if (j * j >= i) {
      println(j);
      break;
    }
  }
}
var end = clock();
print(end - start);

var a;
for (a = 1; a < 10; a = a + 1) {
  if (a == 3) continue;
  if (a == 6) break;
  println(a);
}
println(a);
