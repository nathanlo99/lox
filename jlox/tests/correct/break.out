The ceiling of square root of 0 is: 0
The ceiling of square root of 1 is: 1
The ceiling of square root of 2 is: 2
The ceiling of square root of 3 is: 2
The ceiling of square root of 4 is: 2
The ceiling of square root of 5 is: 3
The ceiling of square root of 6 is: 3
The ceiling of square root of 7 is: 3
The ceiling of square root of 8 is: 3
The ceiling of square root of 9 is: 3
The ceiling of square root of 10 is: 4
The ceiling of square root of 11 is: 4
The ceiling of square root of 12 is: 4
The ceiling of square root of 13 is: 4
The ceiling of square root of 14 is: 4
The ceiling of square root of 15 is: 4
The ceiling of square root of 16 is: 4
The ceiling of square root of 17 is: 5
The ceiling of square root of 18 is: 5
The ceiling of square root of 19 is: 5
1
2
4
5
6
